src/Npgsql/NpgsqlLargeObjectManager.cs

```diff
@@ -10,10 +10,10 @@ namespace Npgsql
     /// </summary>
     public class NpgsqlLargeObjectManager
     {
-        const int INV_WRITE = 0x00020000;
-        const int INV_READ = 0x00040000;
+        const int InvWrite = 0x00020000;
+        const int InvRead = 0x00040000;
 
-        internal readonly NpgsqlConnection _connection;
+        internal readonly NpgsqlConnection Connection;
```

Comment:
It's a field, not property.

Reply:
Rider isn't complaining about anything :) I can make it a property, but it really doesn't seem to matter much...

Reply:
I would like to follow the standard naming convention. Properties should be in PascalCase, while fields in _camelCase with underscores before (there are `s_` for statics and `t_` for thread statics to be sure).

Reply:
Changed this to a property.

Regardless, we haven't used `s_` and `t_` up to now, I'm not sure these are actually widely-used anymore.